src/vl-map-layer-switcher.js

```diff
@@ -37,13 +41,17 @@ export class VlMapLayerSwitcher extends vlElement(HTMLElement) {
   }
 
   connectedCallback() {
-    this._addClickListeners();
+    this._mapElement.ready.then(() => this._processInputs());
   }
```

Comment:
Je kan de functie eventueel async maken zodat je met await kunt werken.

Reply:
Ready is momenteel een getter waardoor deze niet async gemaakt kan worden. Dit zou een breaking change introduceren voor de afnemers. Is het goed dat we de code as is voorlopig behouden?

Reply:
Ik bedoel de connectedCallback, en dan “await this._mapElement.ready” doen.

Reply:
Dat is mogelijk. Ik zie niet echt een meerwaarde, maar zeker ook geen minderwaarde dus als het daardoor leesbaarder wordt voor jou is dat prima voor mij.

Reply:
Ja het is een detail en ook een beetje smaakgebonden, dus laat gerust zoals het is hoor :)

Reply:
Geen probleem, het is al aangepast (nog niet gepushed).